templates/patterns/icons_with_text/icons_with_text.ui_patterns.yml

```diff
@@ -8,11 +8,13 @@ icons_with_text:
       description: "List of items, each having an 'icon' and a 'text' field"
       preview:
         - icon: "file"
+          size: "s"
           text: "Culture | Innovation"
         - icon: "calendar"
           text: "15-16 November 2019, 08:00 AM"
         - icon: "location"
           text: "Brussels, Belgium"
+          size: "s"
```

Comment:
```suggestion
          size: "xs"
```
as in Event

Reply:
No need for this either, is just the pattern.

Reply:
Yes, so I suggest to show `xs` since it's used in the Event. Would be nice to group 2 items with icon `xs` and then 2 items with default site to see different difference.